packages/@okta/vuepress-theme-prose/layouts/AuthorizationLayout.vue

```diff
@@ -24,6 +24,20 @@ export default {
     Header: () => import("../components/Header"),
     SmartLink: () => import("../components/SmartLink"),
   },
+  data() {
+    return {
+      appContext: {
+        isTreeNavMobileOpen: false,
+        isInMobileViewport: false,
+        treeNavDocs: []
+      }
+    };
+  },
+  provide() {
+    return {
+      appContext: this.appContext,
+    };
+  },
```

Comment:
Adding here in case you missed it on slack. This is what's breaking the test. Reason is unknown.